Python/Product/Analysis/Parsing/CodeFormattingOptions.cs

```diff
@@ -236,6 +236,11 @@ public sealed class CodeFormattingOptions {
         [CodeFormattingDefaultValue(true)]
         public bool BreakMultipleStatementsPerLine { get; set; }
 
+        [CodeFormattingExample("foo.__bar", "foo._A__bar")]
+        [CodeFormattingCategory(CodeFormattingCategory.Statements)]
+        [CodeFormattingDescription("UseVerbatimImageShort", "UseVerbatimImageLong")]
+        [CodeFormattingDefaultValue(true)]
+        public bool UseVerbatimImage { get; set; }
```

Comment:
This probably shouldn't be exposed to users. Not sure whether we support any attribute for that though.